apps/fluent-tester/package.json

```diff
@@ -21,13 +21,15 @@
   },
   "dependencies": {
     "@fluentui/react-native": ">=0.18.3 <1.0.0",
+    "@fluentui-react-native/experimental-text": ">=0.1.0 <1.0.0",
     "@fluentui-react-native/framework": ">=0.3.7 <1.0.0",
     "@fluentui-react-native/stack": ">=0.2.16 <1.0.0",
     "@fluentui-react-native/interactive-hooks": ">=0.5.2 <1.0.0",
-    "@uifabricshared/theming-react-native": ">=0.8.2 <1.0.0",
-    "@uifabricshared/theme-registry": ">=0.3.71 <1.0.0",
+    "@fluentui-react-native/default-theme": ">=0.2.1 <1.0.0",
+    "@fluentui-react-native/theme": ">=0.2.1 <1.0.0",
+    "@fluentui-react-native/theme-types": ">=0.2.1 <1.0.0",
+    "@fluentui-react-native/win32-theme": ">=0.3.1 <1.0.0",
```

Comment:
Why isn't it a problem that this shared package.json has win specific dependencies?

Reply:
win32-theme isn't 100% win32 specific. It is effectively a theme that maps the Office Palette structure to the Fluent theme structure. Now the office theming native module is only implemented on win32, so it ends up falling back to the JS only implementation for all other platforms.

I was tempted to call this office-theme, and long-term that may make sense, but I'm unclear on which platforms use the office palette and which don't, so for right now using the more scoped naming.

This change ultimately creates a hybrid theme in the tester where it can dynamically bring in layers. It starts with default-theme, optionally specifying light and dark, optionally applying the office theme as a layer, then optionally applying a brand. Long-term we can extend this with more options, this is a start.